examples/gitops/argocd/main.tf

```diff
@@ -91,6 +91,13 @@ module "eks_blueprints_kubernetes_addons" {
       repo_url           = "https://github.com/aws-samples/eks-blueprints-workloads.git"
       add_on_application = false
     }
+    /* other_workloads = {
+      path               = "kustomize-envs/overlays/dev"
+      # At the moment there are no kustomize examples in the aws-samples repo.
+      repo_url           = "https://github.com/aws-samples/eks-blueprints-workloads.git"
+      add_on_application = false
+      type               = "kustomize"
+    } */
```

Comment:
Please remove this block from here. The doc update should be enough. Maybe add a link for a [sample kustomize app](https://github.com/argoproj/argocd-example-apps/tree/master/kustomize-guestbook) in the docs for reference.

Reply:
Done @askulkarni2 . Let me know if there is anything else that you want/need before it can be merged.